docs/patterns.md

```diff
@@ -88,7 +88,27 @@ There are a variety of different patterns for restricting access that you can fo
 * Specific access rules for every command (Role-based Access Control)
 * Blacklisting/whitelisting commands in specific rooms
 
-### Simple (full vs restricted access)
+### Restricting access to all commands
+
+To completely prevent Hubot from operating for a given user, use a receive middleware:
```

Comment:
It was, as I think it's the most basic receive middleware, but you're right that it's weird to duplicate it. On further thought, I removed it from the patterns document. I'm not sure it's a popular enough use case to call a "pattern"; hubot's gone years without it, after all.